protocol-designer/src/assets/localization/en/shared.json

```
@@ -57,27 +60,43 @@
       "body6": "Please note that in order to run the updated protocol on your robot successfully, the OT-2 App and robot are required to be updated to version 3.10.0 or higher."
     }
   },
+  "message_exact_labware_match": "This labware is identical to one you have already uploaded.",
+  "message_invalid_json_file": "Protocol Designer only accepts custom JSON labware definitions made with our Labware Creator",
+  "message_not_json": "Protocol Designer only accepts custom JSON labware definitions made with our Labware Creator. This is not a .json file!",
```

```suggestion
  "message_not_json": "Protocol Designer only accepts JSON files.",
```
something about that exclamation point in the original seemed a little mean 😅